src/gcodeExport.cpp

```diff
@@ -1027,6 +1035,10 @@ void GCodeExport::writeJerk(double jerk)
         {
             *output_stream << "M207 X";
         }
+        else if (getFlavor() == EGCodeFlavor::REPRAP)
+        {
+            *output_stream << "M566 X" << PrecisionedDouble{2, jerk * 60} << " Y" << PrecisionedDouble{2, jerk * 60} << new_line;
```

Comment:
You should refactor the lines below.
The lines 1046-1048 should be copied into the Repetier case and the else case, but not into the reprap case.